### DISCUSSION TITLE
Will amberelec be ported to 353M / RK3566 ?

### DISCUSSION THREAD

**Discussion by shineangelic** [351ELEC/351ELEC#1016] [Questions & Answers]
I've seen a [similar ](https://github.com/AmberELEC/AmberELEC/discussions/978) question mentioning rk3566 and I was wondering if it will be the same for Anbernic RG353M.

I guess I could install it and it would work like it was on 3326? I love amberelec and since I'm moving from rg351m, I'd stick to it even if it's not yet optimized for newer rockchip.

Am I wrong? 
Thanks

**Comment by shineangelic** [comment]
no answer? ANy news for 3566? Thanks

**Comment by dhwz** [comment]
It won't work unless we support the rk3566 platform.
